src/theme-default/components/DocFooter/index.module.scss

```diff
@@ -38,14 +38,23 @@
 }
 
 .prev-next {
-  border-top: 1px solid var(--island-c-divider-light);
-  padding-top: 24px;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
 }
 
 @media (min-width: 640px) {
+  .prev-next {
+    display: flex;
```

Comment:
why not combine the two block? because they represents the same class

Reply:
sorry,I did not notice.now it has been modified. please review.